kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -90,6 +90,9 @@ module.exports = {
         iconButton: {
           module: require('../vue/icon-button'),
         },
+        textbox: {
```

Comment:
that's something I was wondering myself. I went with no-camel because of the html element that's just "texbox" and [because that's how HTML does it](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/textbox) and that it was treated as [one word by Keen](https://josephuspaye.github.io/Keen-UI/#/ui-textbox)